tutorials/joint_controller.md

```diff
@@ -0,0 +1,685 @@
+\page jointcontrollers Joint Controllers
+
+Gazebo provides three joint controller plugins which are `JointController`, `JointPositionController`, and `JointTrajectoryController`.
+
+Let's see a detailed description of each of them and an example usage to help users select the right joint controller for their usage.
+
+## 1) JointController
+
+- Joint controller which can be attached to a model with a reference to a single joint.
+- Currently, only the first axis of a joint can be actuated.
+
+### Modes of JointController
+
+1) Velocity mode:
+This mode lets the user control the desired joint velocity directly.
+
+2) Force mode:
+A user who wants to control joint velocity using a PID controller can use this mode.
+
+**Note**: This force mode is for the user who looking to manually tune PID gains for velocity control according to a specific use case (e.g. Custom models). For general testing purposes, velocity mode will give the best results.
+
+All the parameters related to this controller can be found \ref gz::sim::systems::JointController "here".
+
+The commanded velocity(cmd_vel) can be published or subscribed at the topic: `/model/<model_name>/joint/<joint_name>/cmd_vel` by default.
+
+Message data type: `Double`
+
+### Example usage
+
+Let's see an example for both modes using a simple model having only one joint.
+
+For controlling joints one would require adding the Gazebo's joint controller plugin to the existing `<model_name>.sdf` file.
+
+1) Save the SDF file in the desired directory or create one
+
+e.g.
+
+```bash
+mkdir gz_tutorial
+cd gz_turtorial
+```
+
+2) In this tutorial we will be using the following SDF file (this is just a slight modification of the original `joint_controller.sdf` [example](https://github.com/gazebosim/gz-sim/blob/gz-sim7/examples/worlds/joint_controller.sdf)).
+
+After changing the directory, name the SDF file as `example.sdf`
+
+- SDF file:
+
+```xml
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="default">
+    <plugin
+      filename="gz-sim-physics-system"
+      name="gz::sim::systems::Physics">
+    </plugin>
+    <plugin
+      filename="gz-sim-scene-broadcaster-system"
+      name="gz::sim::systems::SceneBroadcaster">
+    </plugin>

```

Comment:
It is not necessary to have these plugins here. I suggest removing them so that the default world plugins (including these two and some others) could be loaded.